src/ddmd/dclass.d

```diff
@@ -179,6 +179,12 @@ struct ClassFlags
     alias hasDtor = Enum.hasDtor;
 }
 
+enum ubyte IScom = 1;     // mask for COM class (meaning it derives from IUnknown)
+enum ubyte IScpp = 2;     // mask for a C++ interface
+enum ubyte ISobjc = 4;    // mask for an Objective-C class/interface
+enum ubyte ISscope = 8;   // mask for a scope class
+enum ubyte ISanon = 16;   // mask for an anonymous class
```

Comment:
Can you make this an enum, like https://github.com/dlang/dmd/blob/db221adb6a865ca0f3b1c24d922d9c3d5742a097/src/ddmd/blockexit.d#L37 ?

Reply:
is it going to use a single ubyte that way?

Reply:
if yes, of course I can

Reply:
Done